TechLadies/irrc-madwish
# Feature/irrc 119

# Login


[IRRC-118](https://bootcamp-irrc.atlassian.net/browse/IRRC-118)
[IRRC-119](https://bootcamp-irrc.atlassian.net/browse/IRRC-119)


## Description

This change completes the login implementation. There are some minor bugfixes, and I also removed calls to `router.go()`, which I think are unnecessary (and now cause the user to be logged out). `handleResponse` checks if the request failed because the token expired, in which case the user should be logged out.


## Test Steps

1. Set the following values in your `.env` file in `/backend`.
    1. ENABLE_AUTH=true
    2. ENABLE_SIGNUP=true
2. If you haven't already, create a User via the `/users/signup` endpoint.
3. Run the project and login with your user. All data should be shown as normal.
